doc/content/verification_and_validation/ver-1b.md

```diff
@@ -47,11 +49,12 @@ taken as unity, and the distance, x, was taken as 0.5 in this comparison.
 TMAP8 initially under predicts but the results match well subsequently. Comparison
 results are shown in []
```

Comment:
Can you please fix the reference in the above paragraph? The equation and table numbers are not updated. 
Maybe remove the table cross-reference since the documentation does not have a table.